Replicator/Pusher+Revs.cc

```diff
@@ -85,21 +85,30 @@ namespace litecore::repl {
                                                       &c4err);
             if (encryptedRoot) {
                 root = encryptedRoot;
-            } else if (c4err.domain == WebSocketDomain && c4err.code == 503) {
-                // This is treated as a transient network glitch, we lift it to the replicator
-                // to handle. The replicator will be taken to offline and restarted after a certain
-                // wait time.
-                onError(c4err);
-                return;
             } else {
+                // Error: we don't get the encrypted body.
+                // If the encryptor has not specified an error, we assign
+                // the following error.
+                if (!c4err) {
+                    c4err = {LiteCoreDomain, kC4ErrorCrypto};
+                }
```

Comment:
I think you forgot to remove the same logic in line 109.

Reply:
Good eye!